src/bgw/worker.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+
+#ifndef BGW_WORKER_H
+#define BGW_WORKER_H
+
+#include <postgres.h>
+
+#include <postmaster/bgworker.h>
+
+/**
+ * Parameters to background workers.
+ *
+ * Do not add data here that cannot be simply copied to the background worker
+ * using memcpy(3). If it is necessary to add fields that cannot simply be
+ * copied, we need to start using the send and recv functions for the types.
+ *
+ * Only one of `job_id` and `ttl` is passed currently, with `job_id` being used
+ * for normal tests and `ttl` being used for tests.
```

Comment:
Is `normal tests` and `tests` correct? 

```suggestion
 * for normal jobs and `ttl` being used for tests.
```

Reply:
No, it was a typo.